trees.c

```diff
@@ -674,12 +674,9 @@ void Z_INTERNAL zng_tr_flush_block(deflate_state *s, char *buf, uint32_t stored_
         opt_lenb = static_lenb = stored_len + 5; /* force a stored block */
     }
 
-#ifdef FORCE_STORED
-    if (buf != NULL) { /* force stored block */
-#else
     if (stored_len+4 <= opt_lenb && buf != NULL) {
         /* 4: two words for the lengths */
-#endif
+
```

Comment:
No need for empty line, both comment blocks refer to parts of same statement, contents of comment blocks should be merged.

Reply:
Fixed